test/wot-apisecurityscheme/mapping.yarrrml

```diff
@@ -0,0 +1,26 @@
+sources:
+  people-source:
+    access: "http://example.org/file.xml"
+    contentType: "application/xml"
+    operationType: read
+    type: wot
+    referenceFormulation: xpath
+    iterator: "/"
+    security:
+      - type: apikey
```

Comment:
Where does this come from?

Reply:
Not sure what you mean... 
`apikey` maps onto `APISecurityScheme` of WoT Security.